src/Kovan-DssSpell.sol

```diff
@@ -13,73 +13,208 @@
+        DssExecLib.setContract(MCD_END,  "vat", MCD_VAT);
+        DssExecLib.setContract(MCD_END,  "cat", MCD_CAT);
+        DssExecLib.setContract(MCD_END,  "dog", MCD_DOG);
+        DssExecLib.setContract(MCD_END,  "vow", MCD_VOW);
+        DssExecLib.setContract(MCD_END,  "pot", MCD_POT);
+        DssExecLib.setContract(MCD_END, "spot", MCD_SPOT);
+
+        // Authorize the new END in contracts
+        DssExecLib.authorize(MCD_VAT, MCD_END);
+        DssExecLib.authorize(MCD_CAT, MCD_END);
+        DssExecLib.authorize(MCD_DOG, MCD_END);
+        DssExecLib.authorize(MCD_VOW, MCD_END);
+        DssExecLib.authorize(MCD_POT, MCD_END);
+        DssExecLib.authorize(MCD_SPOT, MCD_END);
+
+        // Set wait time in END
+        Fileable(MCD_END).file("wait", EndLike(MCD_END_OLD).wait());
+
+        // Deauthorize the old END in contracts
+        DssExecLib.deauthorize(MCD_VAT, MCD_END_OLD);
+        DssExecLib.deauthorize(MCD_CAT, MCD_END_OLD);
+        DssExecLib.deauthorize(MCD_VOW, MCD_END_OLD);
+        DssExecLib.deauthorize(MCD_POT, MCD_END_OLD);
+        DssExecLib.deauthorize(MCD_SPOT, MCD_END_OLD);
+
+        // Deauthorize the old END from all the FLIPS
+        // Authorize the new END in all the FLIPS
+        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY_OLD).list();
+        for (uint256 i = 0; i < ilks.length; i++) {
+            bytes32 ilk = ilks[i];
+
+            address flip = OldRegistryLike(ILK_REGISTRY_OLD).flip(ilk);
+            DssExecLib.deauthorize(flip, MCD_END_OLD);
+            DssExecLib.authorize(flip, MCD_END);
+
+            try DssExecLib.removeReaderFromOSMWhitelist(OldRegistryLike(ILK_REGISTRY_OLD).pip(ilk), MCD_END_OLD) {} catch {}
+            try DssExecLib.addReaderToOSMWhitelist(OldRegistryLike(ILK_REGISTRY_OLD).pip(ilk), MCD_END) {} catch {}
+        }
+
+        // ------------------  ESM  ------------------
+
+        // Authorize new ESM to execute in new END
+        DssExecLib.authorize(MCD_END, MCD_ESM_BUG);
+        DssExecLib.authorize(MCD_END, MCD_ESM_ATTACK);
+
+        // Authorize new ESM to execute in new VAT
+        DssExecLib.authorize(MCD_VAT, MCD_ESM_ATTACK);
+
+        // ------------------  DOG  ------------------
+
+        // Set VOW in the DOG
+        DssExecLib.setContract(MCD_DOG, "vow", MCD_VOW);
+
+        // Authorize DOG can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_DOG);
+
+        // Authorize DOG can access to VOW
+        DssExecLib.authorize(MCD_VOW, MCD_DOG);
+
+        Fileable(MCD_DOG).file("Hole", 10 * THOUSAND * RAD);
+
+
+        // --------------  CLIPPER_MOM  --------------
+
+        ClipperMomAbstract(CLIPPER_MOM).setAuthority(DssExecLib.getChangelogAddress("MCD_ADM"));
+
+        // TODO: Tolerance currently set to 50%. Verify governance parameter.
+        //   n.b. 600000000000000000000000000 == 40% tolerance
+        ClipperMomAbstract(CLIPPER_MOM).setPriceTolerance(MCD_CLIP_LINK_A, 500000000000000000000000000);
```

Comment:
Can this be set to `50 * RAY / 100` or something to be more concise?

Reply:
Done. However the final value is still pending to be defined.